lib/active_model/serializer.rb

```diff
@@ -177,6 +177,10 @@ def json_key
       end
     end
 
+    def as_json
+      self.class.adapter.new(self).as_json
```

Comment:
Personally, I'm using `#as_json` on serializer instances outside of controllers in two places on a few projects:

- Testing
  - There's workarounds and such posted in the comments, but it's nice to just do:
     ```ruby
let(:foo)        { create :foo }
let(:serializer) { FooSerializer.new(foo).as_json)

... 
expect(serializer[:some_attribute]).to eq 'something'
....
```

  - It's a lot less code to setup, and makes it easy to test attributes included conditionally or the such.

-  We also use AMS to serialize push messages for Apple/GCM - it happens in a background worker far removed controllers, so all the render / ActionController / ActionPack stuff is far removed - we just have a `PushMessageSerializer` we can `#as_json` right over to Grocer or GCM.

Anyways, food for thought... I wouldn't ever use the `#as_json` directly in a controller.